index.html

```diff
@@ -1,65 +1,247 @@
+                    <h2>I am a software Developer</h2>
+                    <p>I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div class="socia-div">
+                    <ul class="social-icons">
+                        <li class="social-icon">
+                            <a href="#">
+                                <i class="fab fa-github"></i>
+                            </a>
+                        </li>
+                        <li class="social-icon">
+                            <a href="#">
+                                <i class="fab fa-linkedin"></i>
+                            </a>
+                        </li>
+                        <li class="social-icon">
+                            <a href="#">
+                                <i class="fab fa-angellist"></i>
+                            </a>
+                        </li>
+                        <li class="social-icon">
+                            <a href="#">
+                                <i class="fab fa-twitter"></i>
+                            </a>
+                        </li>
+                        <li class="social-icon">
+                            <a href="#">
+                                <i class="fab fa-medium"></i>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </section>
+
+            <!--Recent Works Section-->
+            
+            <section class="my-recent-works">
+                <div class="recent-works-heading">
+                <h2 class="recent-works-title">My Recent Works</h2>
+                <hr class="header-underline">
+                </div>
+
+                <div class="my-projects">
+                    <div class="multipost-stories">
+                        <img src="./img/multipost-Img.jpg" alt="multi-post">
+                        <div>
+                            <h3>Multi-Post Stories</h3>
+                            <p>
+                                A daily selection of privately personalized reads; 
+                                no accounts or sign-ups required. has been the industry's 
+                                standard dummy text ever since the 1500s, when an unknown 
+                                printer took a standard dummy text.
+                            </p>
+                            <ul class="automation">
+                                <li class="css">css</li>
+                                <li class="html">html</li>
+                                <li class="bootstrap">bootstrap</li>
+                                <li class="ruby">Ruby</li>
+                            </ul>
+                            <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] great job ! your website looks awesome and clean! kindly add the button styling: Active, Hover, and Pressed like shown in figma

![image](https://user-images.githubusercontent.com/89282221/185761479-952ba3da-f9b3-4538-9e6d-adade252f92f.png)